### PR TITLE
Arabic facades

### PR DESCRIPTION
**Pull request by Mahmoud217TR**
I didn't want to translate the word "stub" to "بذرة" so I left it stub is that ok or is there any other translation suggestions?
also I translated "Real-Time" to "فوري" instead of "أثناء الاستخدام".

### CODE REVIEW

**Comment by karam-mustafa** [comment]
@Mahmoud217TR Hello, 
I suggest translating "stub" into: "النموذج القابل للاستبدال"
and Real-time info: "الوقت الحالي" or "وقت النتفيذ" 
If you had finished, please add the new words into dictionary

File: facades.md [hunk 1]
```diff
@@ -1,259 +1,281 @@
...
+خلال قراءتك لتوثيق لارافيل ستشاهد أمثلة لأكواد تقوم بالتفاعل مع ميزات لارافيل بواسطة "facades". توفر واجهات لارافيل Facades واجهة ستاتيكية (static interface) للصفوف المتوافرة في [حاوي خدمات](/docs/{{version}}/container) التطبيق. تحتوي لارافيل العديد من واجهات Facades التي توفر الوصول لمعظم ميزات لارافيل. 
 
-Laravel facades serve as "static proxies" to underlying classes in the service container, providing the benefit of a terse, expressive syntax while maintaining more testability and flexibility than traditional static methods. It's perfectly fine if you don't totally understand how facades work under the hood - just go with the flow and continue learning about Laravel.
+تعمل واجهات لارافيل Facades بصفتها وكيل ستاتيكي ("static proxies") للصفوف الأساسية في حاوي الخدمة، حيث توفر ميزة الصياغة المعبرة والموجزة مع الحفاظ على قابلية الاختبار ومرونة أكبر من الطرق الستاتيكية (static methods) التقليدية. من الطبيعي جداً عدم فهمك بشكل كامل لكيفية عمل Facades ضمنياً، عليك فقط بالمسايرة واكمال تعلمك عن لارافيل.
```
Comment (karam-mustafa) [inline facades.md]: we will translate class as "كائن"

File: facades.md [hunk 2]
```diff
@@ -1,259 +1,281 @@
...
 
-Laravel facades serve as "static proxies" to underlying classes in the service container, providing the benefit of a terse, expressive syntax while maintaining more testability and flexibility than traditional static methods. It's perfectly fine if you don't totally understand how facades work under the hood - just go with the flow and continue learning about Laravel.
+تعمل واجهات لارافيل Facades بصفتها وكيل ستاتيكي ("static proxies") للصفوف الأساسية في حاوي الخدمة، حيث توفر ميزة الصياغة المعبرة والموجزة مع الحفاظ على قابلية الاختبار ومرونة أكبر من الطرق الستاتيكية (static methods) التقليدية. من الطبيعي جداً عدم فهمك بشكل كامل لكيفية عمل Facades ضمنياً، عليك فقط بالمسايرة واكمال تعلمك عن لارافيل. 
 
-All of Laravel's facades are defined in the `Illuminate\Support\Facades` namespace. So, we can easily access a facade like so:
+كل واجهات لارافيل Facades معرفة ضمن فضاء الأسماء `Illuminate\Support\Facades`. أي أنه يمكننا وبسهولة الوصول لـ Facades بالأسلوب التالي:
```
Comment (karam-mustafa) [inline facades.md]: convert "فضاء" to "نطاق" and add it to the dictionary please

File: facades.md [hunk 3]
```diff
@@ -1,259 +1,281 @@
...
+واستكمالاً لواجهات Facades تقدم لارافيل عدة توابع مساعدة متنوعة لتجعل التعامل مع الميزات العامة في لارافيل أكثر سهولة. قد تتعامل مع بعض التوابع المساعدة الشائعة مثل `view`، `response`، `url`، `config` وغيرها المزيد. كل تابع مساعد مقدم من لارافيل يتم توثيقه مع الميزة التي يقدمها، وبكل الأحوال تتوفر لائحة كاملة في [التوثيق المخصص للتوابع المساعدة](/docs/{{version}}/helpers) .
 
-For example, instead of using the `Illuminate\Support\Facades\Response` facade to generate a JSON response, we may simply use the `response` function. Because helper functions are globally available, you do not need to import any classes in order to use them:
+على سبيل المثال، بدلاً من استخدام الـ  Facade التالي `Illuminate\Support\Facades\Response` لانشاء رد من النوع JSON يمكنك ببساطة استخدام التابع المساعد `response`. ولأن التوابع المساعدة متوفرة على نطاق التطبيق (globally available)، لست بحاجة لاستيراد (import) أي صفوف لاستخدمها:
```
Comment (karam-mustafa) [inline facades.md]: I think it is better to add (response) after "رد"

File: facades.md [hunk 4]
```diff
@@ -1,259 +1,281 @@
...
-
-The `Facade` base class makes use of the `__callStatic()` magic-method to defer calls from your facade to an object resolved from the container. In the example below, a call is made to the Laravel cache system. By glancing at this code, one might assume that the static `get` method is being called on the `Cache` class:
+## كيفية عمل واجهات Facades
 
-    <?php
+في تطبيق لارافيل، واجهة Facade هي صف يوفر الوصول لكائن من حاوي الخدمات. حيث أن الآلية التي تسمح بذلك موجودة في الصف `Facade`. إن واجهات لارافيل Facades وأي واجهات Facades تقوم بإنشائها، سوف ترث من الصف الأساسي `Illuminate\Support\Facades\Facade`. 
 
-    namespace App\Http\Controllers;
+الصف الأساسي `Facade` يستخدم الطريقة السحرية (magic-method) `()callStatic__` لتحويل الاستدعاءات من واجهة Facade لكائن يتم الحصول عليه من حاوي الخدمات. في المثال التالي، سيتم استدعاء نظام الذاكرة المؤقت (cache system) الخاص بلارافيل. عند القاء نظرة على هذا الكود قد تظن بأن الطريقة الستاتيكية `get` يتم استدعائها على الصف `Cache`:
```
Comment (karam-mustafa) [inline facades.md]: I don't know if the "الطريقة السحرية" is a scientific definition, instead we can say "التوابع الجاهزة" or "تابع الكائن الاساسي"

File: facades.md [hunk 5]
```diff
@@ -1,259 +1,281 @@
...
-        protected static function getFacadeAccessor() { return 'cache'; }
+        $user = Cache::get('user:'.$id);
+
+        return view('profile', ['user' => $user]);
     }
+}
+```
+لاحظ في أعلى الملف بأننا نقوم باستيراد (importing) واجهة Facade التالية `Cache`. حيث تقوم واجهة Facade بالعمل كوكيل للوصول للاستخدام (implementation) الخاص بالواجهة `Illuminate\Contracts\Cache\Factory`. حيث أن أي استدعاءات نقوم بها باستخدام واجهة Facade سيتم تمريرها للنسخة (instance) الضمنية في خدمة الذاكرة المؤقتة في لارافيل. 
+
+اذا قمنا بإلقاء نظرة على الصف `Illuminate\Support\Facades\Cache` ستجد بأنه لا وجود للطريقة الستاتيكية `get`:
```
Comment (karam-mustafa) [inline facades.md]: the word "ستاتيك" is accepted, please add it into dictionary

**Review by karam-mustafa — CHANGES_REQUESTED** [review]

**Comment by Mahmoud217TR** [comment]
Consider it done, no worries.

File: facades.md (hunk 1 above)
Comment (Mahmoud217TR) [reply facades.md]: you mean instead of "الصفوف الأساسية" we should use "كائنات أساسية" ?

Comment (Mahmoud217TR) [reply facades.md]: or should I replace all "صفوف" to "كائنات"

Comment (Mahmoud217TR) [reply facades.md]: other modifications are ready, waiting for confirmations about classes

Comment (karam-mustafa) [reply facades.md]: yes replace all words that contains "صف او صفوف" into "كائن او كائنات"

Comment (Mahmoud217TR) [reply facades.md]: Ok I'm on it.

**Comment by Mahmoud217TR** [comment]
I guess this is done.
Ready for any other suggestions.

**Pull request merged**